go/skynet.go

```diff
@@ -4,27 +4,28 @@ import "fmt"
 import "time"
 
 func skynet(c chan int, num int, size int, div int) {
```

Comment:
You could also remove the extra types. I.e. `func skynet(c chan int, num, size, div int)`